testsuite/pytests/test_calcium.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# test_calcium.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test models with calcium concentration.
+This set of tests verify the behavior of the calcium concentration in models
+that inherit from the strutural plasticity node class in the kernel.
+"""
+
+import nest
+import numpy as np
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def reset_kernel(self):
```

Comment:
```suggestion
def reset_kernel():
```
Missed this error from a merge conflict in the review.